packages/elements/src/components/ino-dialog/ino-dialog.scss

```diff
@@ -9,16 +13,35 @@
    * @prop --ino-dialog-header-height: height of the header section
    * @prop --ino-dialog-content-height: height of the content section
    * @prop --ino-dialog-footer-height: height of the footer section
+   * @prop --ino-dialog-border-radius: border-radius of the dialog
+   * @prop --ino-dialog-border: border shorthand of the dialog
+   * @prop --ino-dialog-box-shadow: box-shadow of the dialog
+   * @prop --ino-dialog-overlay-background-color: background color of the overlay
    */
   --dialog-background-color: var(--ino-dialog-background, #fff);
```

Comment:
`--ino-dialog-background-color` does not get picked up (because of `--dialog-background-color: var(--ino-dialog-background, #fff);`) -> `...-color` is missing